src/components/post/Post.js

```diff
@@ -72,18 +72,17 @@ const Post = (props) => {
     }
   };
 
-  // adds like to post
-  const like = (postID) => {
-    setLiked(true);
-    setLikes(likes + 1);
-    props.like(postID);
-  };
-
-  // removes like from post
-  const unlike = (postID) => {
-    setLiked(false);
-    setLikes(likes - 1);
-    props.unlike(postID);
+  // handles liking and unliking a post (makes call based on postID being passed in)
+  const onClickLikesHandler = (postID) => {
```

Comment:
I'm not sure where this postID is coming from since this is an event but everything else looks good

Reply:
I believe you are both correct. It is not needed. Just tested in without it and seems to not affect anything. Good Catch!